docs/units-and-global-variables.rst

```diff
@@ -115,14 +115,6 @@ Block and Transaction Properties
     You can only access the hashes of the most recent 256 blocks, all other
     values will be zero.
 
-.. note::
-    The function ``blockhash`` was previously known as ``block.blockhash``, which was deprecated in
-    version 0.4.22 and removed in version 0.5.0.
-
-.. note::
-    The function ``gasleft`` was previously known as ``msg.gas``, which was deprecated in
-    version 0.4.21 and removed in version 0.5.0.
-
```

Comment:
Ah, wait so they are here after all, just not as a single note but under each entry separately. Then it was actually fine. Sorry for that. We should go back to the way you did it originally.

Reply:
Alright